deploy/webhook/member-operator-webhook.yaml

```diff
@@ -0,0 +1,107 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: member-operator-webhook
+objects:
+- apiVersion: scheduling.k8s.io/v1
+  kind: PriorityClass
+  metadata:
+    name: sandbox-users-pods
+    labels:
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  value: -10
+  globalDefault: false
+  description: "Priority class for pods in users' namespaces"
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: member-operator-webhook
+    namespace: ${NAMESPACE}
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  spec:
+    ports:
+    - port: 443
+      targetPort: 8443
+    selector:
+      app: member-operator-webhook
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: member-operator-webhook
+    namespace: ${NAMESPACE}
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: member-operator-webhook
+    template:
+      metadata:
+        name: member-operator-webhook
+        labels:
+          app: member-operator-webhook
+      spec:
+        containers:
+        - name: mutator
+          image: ${IMAGE}
+          command:
+          - member-operator-webhook
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 500m
+              memory: 500Mi
+            requests:
+              cpu: 250m
+              memory: 128Mi
+          volumeMounts:
+          - name: webhook-certs
+            mountPath: /etc/webhook/certs
+            readOnly: true
+        volumes:
+        - name: webhook-certs
+          secret:
+            secretName: webhook-certs
+- apiVersion: admissionregistration.k8s.io/v1
+  kind: MutatingWebhookConfiguration
+  metadata:
+    name: member-operator-webhook
+    labels:
+      app: member-operator-webhook
+      toolchain.dev.openshift.com/provider: codeready-toolchain
+  webhooks:
+  - name: users.pods.webhook.sandbox
+    admissionReviewVersions:
+      - v1
+    clientConfig:
+      caBundle: ${CA_BUNDLE}
+      service:
+        name: member-operator-webhook
+        namespace: ${NAMESPACE}
+        path: "/mutate-users-pods"
+        port: 443
+    matchPolicy: Equivalent
+    rules:
+    - operations: ["CREATE", "UPDATE"]
+      apiGroups: [""]
+      apiVersions: ["v1"]
+      resources: ["pods"]
+      scope: "Namespaced"
+    sideEffects: None
+    timeoutSeconds: 5
+    reinvocationPolicy: Never
+    failurePolicy: Ignore
+    namespaceSelector:
+      matchLabels:
+        toolchain.dev.openshift.com/provider: codeready-toolchain
+parameters:
+- name: NAMESPACE
+  value: 'toolchain-member-operator'
+- name: IMAGE
+  value: quay.io/matousjobanek/member-operator-webhook:latest
```

Comment:
Is it just temporary? We should already have an image in the codeready org, shouldn't we?

Reply:
:ghost: zombie - let me fix that
and no - we don't have the image in the quay repo yet - I need to modify the CD script first :-)  That's the next step, but should be pretty easy

Reply:
fixed: 0c3135c